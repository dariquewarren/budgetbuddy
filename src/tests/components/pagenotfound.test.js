import React from 'react'
import {shallow} from 'enzyme'
import NotFoundPage from '../../components/PageNotFound'

test('page not found component', ()=>{
    const wrapper =  shallow(<NotFoundPage/>)
    expect(wrapper.props()).toMatchSnapshot()
})