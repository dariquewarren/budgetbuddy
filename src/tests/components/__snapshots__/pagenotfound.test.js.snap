// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`page not found component 1`] = `
Object {
  "children": Array [
    "404 page not found 404 - ",
    <Link
      replace={false}
      to="/"
    >
       go home
    </Link>,
  ],
}
`;
